wrapper/Ada/tls_client.adb

```diff
@@ -276,6 +335,15 @@ package body Tls_Client with SPARK_Mode is
          return;
       end if;
 
+      if not (Ada.Directories.Exists (CERT_FILE) and then
```

Comment:
Is this the best way to gate using PSK vs loading key/cert? A build option or run-time argument seems like it would be ideal (if possible). 

Note: It is possible to set the PSK callback regardless, assuming NO_PSK is not set when building wolfSSL.